internal/rootcoord/root_coord.go

```diff
@@ -2850,11 +2897,21 @@ func (c *Core) CreateCredential(ctx context.Context, credInfo *internalpb.Creden
 	log.Debug("CreateCredential", zap.String("role", typeutil.RootCoordRole),
 		zap.String("username", credInfo.Username))
 
+	usersInfo, err := c.MetaTable.ListCredentialUsernames()
```

Comment:
It's the same. because the operation is in the etcd level and I think it's unnecessary. If in the future, it has more scenes which need to `count`, the `count` will be more useful.

Reply:
OK. works for me if the users number will not be to large.